src/lib/radial-chart/radial-chart.js

```diff
@@ -245,10 +245,6 @@ class RadialChart extends React.Component {
       return null;
     }
 
-    const {innerWidth, innerHeight} = getInnerDimensions(
```

Comment:
it's not broken, it was just being used incorrectly. Pie charts are by their nature centered in their containers and having their sizes modified modifying the range of the slices. The old situation lead to stuff like this 

https://cloud.githubusercontent.com/assets/413395/21851356/da72eab8-d862-11e6-972e-9bf2ae89ace0.png

which was not ideal